lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -316,7 +316,7 @@ void LCIStrategicPlugin::handleCruisingUntilStop(const cav_srvs::PlanManeuversRe
   resp.new_plan.maneuvers.push_back(composeStopAndWaitManeuverMessage(
     new_ts_params.x2_, traffic_light_down_track, new_ts_params.v2_, case_8_crossed_lanelets.front().id(),
     case_8_crossed_lanelets.back().id(), ros::Time(new_ts_params.t2_),
-    current_state.stamp + ros::Duration(config_.min_maneuver_planning_period), decel_rate));
+    ros::Time(new_ts_params.t2_) + ros::Duration(config_.min_maneuver_planning_period), decel_rate));
```

Comment:
What is t2_ here?

Reply:
t2 is the time the vehicle should arrive at x2 which is when the stopping maneuever should start. So previously there was inconsistency in the inputs where it used correct starting time t2, but when calculating ending time, it used current_time + maneuever_planning_period instead of t2_maneuever_planning_period